library/common/config/config.cc

```diff
@@ -24,6 +24,15 @@ const char* native_filter_template = R"(
             typed_config: {{ native_filter_typed_config }}
 )";
 
+const char* dns_template = R"(
+                typed_dns_resolver_config:
+                  name: envoy.network.dns_resolver.cares
+                  typed_config:
+                    "@type": type.googleapis.com/envoy.extensions.network.dns_resolver.cares.v3.CaresDnsResolverConfig
+                    resolvers: {{ dns_resolvers }}
+                    use_resolvers_as_fallback: {{ dns_use_resolvers_as_fallback }}
+)";
```

Comment:
Is it possible to specify a no-op default and override so we can avoid introducing new interpolation?